lib/engine/round/operating.rb

```diff
@@ -42,9 +42,11 @@ class Operating < Base
         company: 'Companies',
       }.freeze
 
-      def initialize(entities, game:, round_num: 1)
+      def initialize(entities, game:, round_num: 1, ebuy_pres_swap: true, ebuy_other_value: true)
```

Comment:
let's change this to **opts

and 48 will be @ebuy_pres_swap = opts[:ebuy_pres_swap].nil? ? opts[:ebuy_pres_swap] : true

since i think there will be many OR options